src/webRTCEndpoint.ts

```diff
@@ -545,6 +563,36 @@ export class WebRTCEndpoint extends (EventEmitter as new () => TypedEmitter<Requ
         break;
       }
 
+      case "trackEncodingDisabled": {
```

Comment:
We might want to store this info in TrackContext too? cc @kamil-stasiak

Reply:
Yes, we would like to have this, but we could add it in the next PR